sidestep_engine/core/lora_module.py

```
@@ -384,42 +376,34 @@ def training_step(self, batch: Dict[str, torch.Tensor]) -> torch.Tensor:
-                    )
-                else:
-                    t, r = sample_timesteps(
-                        batch_size=bsz,
-                        device=self.device,
-                        dtype=self.dtype,
-                        data_proportion=self._data_proportion,
-                        timestep_mu=self._timestep_mu,
-                        timestep_sigma=self._timestep_sigma,
-                        use_meanflow=False,
-                    )
+                t, r = sample_timesteps(
```

Ruff (0.15.5)

[warning] Unpacked variable `r` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)